packages/block-library/src/file/inspector.js

```diff
@@ -26,6 +40,32 @@ export default function FileBlockInspector( {
 	return (
 		<>
 			<InspectorControls>
+				{ href.endsWith( '.pdf' ) && (
+					<PanelBody title={ __( 'PDF settings' ) }>
+						<ToggleControl
+							label={ __( 'Show inline embed' ) }
+							help={
+								displayPreview
+									? __(
+											"Note: Most phone and tablet browsers won't display embedded PDFs."
+									  )
+									: null
+							}
+							checked={ displayPreview }
```

Comment:
To avoid errors for existing files.
```suggestion
							checked={ !! displayPreview }
```

Reply:
I didn't manage to trigger this error, but it's a good safeguard to put in place, anyway.